src/Query/Filter/MultiTextField.ts

```diff
@@ -0,0 +1,53 @@
+import type { Task } from '../../Task';
+import type { IStringMatcher } from '../Matchers/IStringMatcher';
+import { TextField } from './TextField';
+import type { Filter } from './Filter';
+
+/**
+ * MultiTextField is an abstract base class to help implement
+ * all the filter instructions that act on multiple string values
+ * such as the tags.
+ */
+export abstract class MultiTextField extends TextField {
+    public abstract fieldNameSingular(): string;
+
+    public fieldNamePlural(): string {
+        return this.fieldNameSingular() + 's';
+    }
+
+    public fieldName(): string {
+        return `${this.fieldNameSingular()}/${this.fieldNamePlural()}`;
+    }
+
+    protected fieldPattern(): string {
+        return `${this.fieldNameSingular()}|${this.fieldNamePlural()}`;
+    }
+
+    protected operatorPattern(): string {
+        return `${super.operatorPattern()}|include|do not include`;
+    }
+
+    /**
+     * Returns the field's value, or an empty string if the value is null
+     * @param task
+     * @public
+     */
```

Comment:
This is identical to the comment in the base class, and as such, isn't needed, as JSdoc will show the docs for the base class anyway, so it could be removed.

However, it might be worth adjusting the description to make more sense in this context - like, what it joins the values together with.

Reply:
Good point. Improved the JSDoc comments a bit. Feel free to point out other issues or to make changes directly.